### PR TITLE
change aegis name to official

### PR DESCRIPTION
**Pull request by Takerio**
* **Addressed Issue(s)**: 

* **Server Mode**: 
RE

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_db_etc.yml [hunk 1]
```diff
@@ -4213,6 +4213,7 @@ Body:
       DropEffect: CLIENT
     Script: |
       skill "TF_DOUBLE",1;
+      bonus bDoubleRate,5;
```
Comment (Atemo) [inline db/re/item_db_etc.yml:4216]:
```suggestion
```

File: db/re/item_db_etc.yml [hunk 2]
```diff
@@ -42879,6 +42880,7 @@ Body:
     SubType: Enchant
     Script: |
       skill "TF_DOUBLE",3;
+      bonus bDoubleRate,15;
```
Comment (Atemo) [inline db/re/item_db_etc.yml:42883]:
```suggestion
```

**Review by Atemo — CHANGES_REQUESTED** [review]

**Comment by Akkarinage** [comment]
@Takerio can you update your Pull Request please.

In future, you need to create a new branch before doing any work and creating a Pull Request, as we can't remotely update your master branch.

**Comment by Lemongrass3110** [comment]
No update since 3 months.
Most likely already most of it was covered by now.

**Pull request closed without merging**